src/shared/configUtils.ts

```diff
@@ -5,14 +5,27 @@
  * For full license text, see LICENSE.txt file in the repo root or https://opensource.org/licenses/BSD-3-Clause
  */
 
+/* eslint-disable @typescript-eslint/no-unsafe-assignment */
+/* eslint-disable @typescript-eslint/no-unsafe-call */
+/* eslint-disable @typescript-eslint/no-unsafe-member-access */
+
 import { Workspace } from '@lwc/lwc-dev-server';
 import { CryptoUtils, SSLCertificateData } from '@salesforce/lwc-dev-mobile-core';
-import { Config, ConfigAggregator } from '@salesforce/core';
+import { Config, ConfigAggregator, Connection } from '@salesforce/core';
 import configMeta, { ConfigVars, SerializedSSLCertificateData } from './../configMeta.js';
 
 export const LOCAL_DEV_SERVER_DEFAULT_PORT = 8081;
 export const LOCAL_DEV_SERVER_DEFAULT_WORKSPACE = Workspace.SfCli;
 
+export class LocalWebServerIdentityData {
+  public identityToken: string;
+  public usernameToServerEntityIdMap: Record<string, string> = {};
+
+  public constructor(token: string) {
```

Comment:
Nit: let the constructor set the map too
```
export class LocalWebServerIdentityData {
  public identityToken: string;
  public usernameToServerEntityIdMap: Record<string, string>;

  public constructor(token: string, map: Record<string, string> = {}) {
    this.identityToken = token;
    this.usernameToServerEntityIdMap = map
  }
}
```

Reply:
As we spoke on slack this is now a `type` instead of `class`.